src/classes/app.py

```diff
@@ -80,7 +88,7 @@ def __init__(self, *args, mode=None):
         # Log some basic system info
         try:
             log.info("------------------------------------------------")
-            log.info("   OpenShot (version %s)" % info.SETUP['version'])
+            log.info(("OpenShot (version %s)" % info.SETUP['version']).center(48))
```

Comment:
Hmm. While that _does_ work, in this one particular case only, I'm sort of inclined to use an intermediate variable. Because boy is that unreadable, compared to a modified version of your previous edit...
```py
app_name_str = "OpenShot (version %s)" % info.SETUP['version']
...
log.info(app_name_str.center(48))
```
Personally I hate the `%`-based (old-style) format strings, and this kind of situation is exactly why. In modern-er python this would be,
```py
log.info("OpenShot (version {})".format(info.SETUP['version']).center(48))
```
which is at least _less_ unreadable (IMHO) — though I'd still probably use an intermediate variable to hold the formatted string.